linera-service/tests/readme_test.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(feature = "aws")]
+use linera_views::test_utils::LocalStackTestContext;
+use once_cell::sync::Lazy;
+use std::io::Write;
+use tempfile::tempdir;
+use tokio::{process::Command, sync::Mutex};
+
+/// A static lock to prevent integration tests from running in parallel.
+static INTEGRATION_TEST_GUARD: Lazy<Mutex<()>> = Lazy::new(|| Mutex::new(()));
```

Comment:
Maybe that should move on level up, to `tests.rs`.

Reply:
Ah, it's even two separate test guards—but the end-to-end tests _do_ conflict with the readme tests, don't they?